test/test-importer.js

```diff
@@ -29,6 +29,17 @@ module.exports = function (repo) {
       done()
     })
 
+    it('bad input', (done) => {
+      const r = 'banana'
+      const i = new Importer(ds)
+      i.on('error', (err) => {
+        expect(err).to.exist
+        done()
+      })
+      i.write({path: '200Bytes.txt', content: r})
+      i.end()
```

Comment:
Let's listen for the `.on('data'` to make sure that the hash matches the right thing